pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
     'copulas>=0.11.0',
     'ctgan>=0.10.0',
     'deepecho>=0.6.0',
-    'rdt @ git+https://github.com/sdv-dev/RDT@main',
+    'rdt @ git+https://github.com/sdv-dev/RDT@issue-865-uint-hypertransformer',
```

Comment:
I will update to point to the main branch once this [PR](https://github.com/sdv-dev/RDT/pull/867) is merged.
Or if the RDT release is done, I could point to the latest RDT version